src/script/cpatch.py

```diff
@@ -613,7 +580,9 @@ def _py_common_job(self, component):
                     exclude_dirs=exclude_dirs,
                     exclude_file_suffixes=exclude_file_suffixes,
                 )
-        return [f"ADD {name} {self._py_site_packages()}"]
+        dlines=[f"ADD {name} tmp_python_common"]
+        dlines.append ("RUN for i in tmp_python_common/*; do find /usr/lib/python*/site-packages -type d -name $(basename $i) -exec cp -frpv $i/* '{}' \;; done && rm -rf tmp_python_common")
```

Comment:
Sorry, but I don't care for this approach. It feels like embedding a full-blown shell script inside a dockerfile inside a python script. On top of that, I intentionally limited the list of python versions to find to list of actual python versions.

Reply:
We used this approach to find the correct location of the binaries and libraries because there is a longer list of sub directories where the files to patch might be located (depending on the distribution used in the base container).  
The alternative approach of running the "ls -d" command inside a container inside a python script would not easily work for these files because there is a longer list of sub directories to search and multiple directories might exist with the ceph binaries only in one of them.  
It seems preferable to use the same approach for finding all files to patch (python, binaries and libraries).  In addition, limiting the directories searched to python versions that exist, means that the script needs continual updates.

Reply:
reverted to original behaviour for python files